cmd/generate/BuildImageDockerfile.template

```diff
@@ -18,6 +18,7 @@ RUN wget https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 &&
 RUN ./get-helm-3 --version v3.11.3 --no-sudo && helm version
 
 RUN GOFLAGS='' go install github.com/mikefarah/yq/v3@latest
+RUN GOFLAGS='' go install knative.dev/test-infra/tools/kntest/cmd/kntest@latest
```

Comment:
Hmm, let me double check it's really required. Because we drop it from release-next at least. 

https://github.com/openshift-knative/serving/commit/1a5133bb2991bfa09804d953e7d7ade34a55622a

Also there was an issue with `$GOPATH/.cache` removal.

Reply:
OK. Let's try without it (I didn't try that actuall, I just saw this in the Dockerfiles on release-v1.15 branch of Serving.

/hold

Reply:
Testing in https://github.com/openshift/release/pull/56274

Reply:
Looks like kntest is not required. The rehearsal job ran [here](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/56274/rehearse-56274-pull-ci-openshift-knative-serving-generate_dockerfiles_1.15_mgencur-415-test-e2e-aws-415/1831389921726697472) and most tests passed. There are a few individual failures which are unrelated.